DataFormats/L1TParticleFlow/src/HPSPFTau.cc

```diff
@@ -34,57 +32,53 @@ ostream& operator<<(ostream& os, const l1t::HPSPFTau& l1PFTau) {
   os << "seed:";
   if (l1PFTau.isChargedPFCandSeeded()) {
     os << " chargedPFCand";
-  } else if (l1PFTau.isPFJetSeeded()) {
+  } else if (l1PFTau.isJetSeeded()) {
     os << " PFJet";
-  } else {
-    cms::Exception ex("InconsistentTau");
-    ex.addContext("Calling HPSPFTau::operator <<");
-    ex.addAdditionalInfo("This tau is not seed by either a chargedPFCand or a PFJet!");
-    throw ex;
-  }
+  } else
+    assert(0);
```

Comment:
Please, don't ever do this.

Can you put back the `cms::Exception` ?

Reply:
Done